app/assets/stylesheets/chat.scss

```diff
@@ -15,7 +15,7 @@
   height: calc(100vh - 56px);
 
   &.live-chat--iossafari {
-    height: calc(100vh - 190px);
+    height: calc(100vh - 56px);
```

Comment:
If you ever want to reference the height of header, you could use [one of our CSS variables](https://github.com/forem/forem/blob/master/app/assets/stylesheets/config/_variables.scss#L10): `var(--header-height)`